chromadb/utils/embedding_functions.py

```diff
@@ -185,10 +186,11 @@ def __call__(self, input: Documents) -> Embeddings:
             # Sort resulting embeddings by index
             sorted_embeddings = sorted(
                 embeddings, key=lambda e: e["index"]
-            )  # type: ignore
+            )
 
             # Return just the embeddings
-            return [result["embedding"] for result in sorted_embeddings]
+            emb1 = [result["embedding"] for result in sorted_embeddings]
+            return cast(Embeddings, emb1)
```

Comment:
`emb1` is a bit weird. Can we just,
```suggestion
            return cast(Embeddings, [result["embedding"] for result in sorted_embeddings])
```